### DISCUSSION TITLE
Stuck on "authenticator app"

### DISCUSSION THREAD

**Discussion by realitylpma** [stoatchat/discussions#467] [Q&A]
Probably a very dumb question but I can't seem to figure out how to get the "authenticator app" screen to do anything.

**Comment by Rexogamer** [comment]
What are you seeing?

**Reply by realitylpma** [reply]
It goes straight to an "authenticator app" pop-up that has a field to enter something and a 'confirm' button

**Reply by Rexogamer** [reply]
Do you have an authenticator app on your phone (e.g. Google Authenticator, Authy, Microsoft Authenticator etc.)?

**Reply by realitylpma** [reply]
Never used any. Just downloaded Google Authenticator to check it out and it seems it would've needed to be pre-setup?

**Reply by Rexogamer** [reply]
Did you scan the QR code in Google Authenticator?

**Reply by realitylpma** [reply]
I'm not getting any, this is all I get:
![image](https://user-images.githubusercontent.com/54695876/203875509-6fb3e465-b34a-4b79-9b56-cc8747ef0221.png)

**Reply by Rexogamer** [reply]
Do you see any six-digit codes in Google Authenticator?

**Reply by realitylpma** [reply]
it's never been setup so i'm stuck on a "set up your first account" screen

**Reply by Rexogamer** [reply]
That's odd - the 2FA prompt should only appear if it's been set up, which would require an authenticator app

**Reply by realitylpma** [reply]
any way to reset it?

**Reply by Rexogamer** [reply]
What is your username on Revolt?

**Reply by realitylpma** [reply]
I think it's realitylpma, I haven't used it much. I'm having the exact same issue with divolt and tried to figure out if they had any relations
